package.json

```diff
@@ -268,6 +268,7 @@
     "test-mssql": "cross-env DIALECT=mssql npm test",
     "test-db2": "cross-env DIALECT=db2 npm test",
     "----------------------------------------- development ---------------------------------------------": "",
+    "fix-commit": "export TMPFILE=$(mktemp) && grep -v '^#' $(git rev-parse --git-dir)/COMMIT_EDITMSG > $TMPFILE && git commit -e -F $TMPFILE",
```

Comment:
The `grep -v` is because otherwise you end up with two commit templates one after the other, which is not the end of the world, but also not great. Better to avoid it.

The tempfile was necessary because not all editors support just piping STDIN into them without any other flags, and some (e.g. vim) horribly break the terminal output when you do, at least on my machine.